doc/classes/adapter.md

```diff
@@ -331,7 +331,7 @@ ___
 
 ▸ **start**(): `void`
 
-*Defined in [scanner/adapters/adapter.ts:34](https://github.com/resin-io-modules/etcher-sdk/blob/e34af4f/lib/scanner/adapters/adapter.ts#L34)*
+*Defined in [scanner/adapters/adapter.ts:34](https://github.com/balena-io-modules/etcher-sdk/blob/1daa03e/lib/scanner/adapters/adapter.ts#L34)*
```

Comment:
Shouldn't the links refer to `master`? e.g. https://github.com/balena-io-modules/etcher-sdk/blob/master/lib/scanner/adapters/adapter.ts#L34 (same for other references below)

Reply:
I guess not so when you browse the docs of older revisions they point to the right place.

Reply:
Right, didn't think about that. Lgtm then